src/EditorFeatures/Core.Wpf/NavigateTo/NavigateToItemProvider.Callback.cs

```diff
@@ -101,6 +101,10 @@ private static string GetKind(string kind)
                     // This should be updated whenever NavigateToItemKind has a record.
                     CodeAnalysis.NavigateTo.NavigateToItemKind.Record
                         => VisualStudio.Language.NavigateTo.Interfaces.NavigateToItemKind.Class,
+                    // VisualStudio.Language.NavigateTo.Interfaces.NavigateToItemKind doesn't have a record struct, fall back to struct.
+                    // This should be updated whenever NavigateToItemKind has a record struct.
```

Comment:
Do we need to route this to the appropriate team?

Not sure if it's the editor team who owns this?

cc: @jinujoseph @olegtk